### PR TITLE
Redirects stderr to stdout when running tests.

### PR DESCRIPTION
**Pull request by amanda-tarafa**
* This avoids PS failing when dotnet test writes to stderr.
  * Will hopefully improve the quality of test failure messages.

See: https://stackoverflow.com/a/20950421/1122643

I'm testing first with Asset, if it fixes the failure we had there, I'll do this for all files.

(Draft while trial and error)

### CODE REVIEW

**Comment by amanda-tarafa** [comment]
FYI @yuyifan-google

**Comment by amanda-tarafa** [comment]
@laljikanjareeya , @skuruppu This fix should make the fail messages less cryptic. I've seen for the Asset samples how we've gone from a Fail <TestName> (similar to what we had in Spanner) to having the whole detail of the failure, exception, stack trace, etc.

**Comment by amanda-tarafa** [comment]
@jskeet This is ready for review, one commit at a time is best.

**Comment by jskeet** [comment]
Great, will look tomorrow

**Comment by laljikanjareeya** [comment]
> @laljikanjareeya , @skuruppu This fix should make the fail messages less cryptic. I've seen for the Asset samples how we've gone from a Fail (similar to what we had in Spanner) to having the whole detail of the failure, exception, stack trace, etc.

@amanda-tarafa  Thanks. This is helpful when something is failing.

File: BuildTools.psm1 [hunk 1]
```diff
@@ -263,7 +263,7 @@ function Run-TestScriptsOnce([array]$Scripts, [int]$TimeoutSeconds,
         Write-Output "$verb $relativePath..."
         $job = Start-Job -ArgumentList $relativePath, $script.Directory, `
             ('.\"{0}"' -f $script.Name) {
-            $ErrorActionPreference = "Stop"
+            $ErrorActionPreference = "Continue"
```
Comment (jskeet) [inline BuildTools.psm1:266]: Is this still required, if we're redirecting stderr to stdout anyway? I'm fine with it, but I thought I'd mention it seeming redundant.

Comment (amanda-tarafa) [reply BuildTools.psm1:266]: Yes, it's actually the combination of both that works, redirecting to stdout doesn't work if this is set to "Stop".

**Pull request merged**